src/simple_controller/proof/liveness.rs

```diff
@@ -74,16 +115,25 @@ proof fn lemma_reconcile_ongoing_leads_to_cm_always_exists(cr: CustomResourceVie
             lift_state(|s: State<SimpleReconcileState>| s.reconcile_state_contains(cr.object_ref()))
```

Comment:
With the current spec we will not be able to prove this lemma. Instead, we will prove a different form of lemma under the assumption of []cr_exists. We can remove this lemma but keeping it for now is also fine.